### ISSUE TITLE
Spring Boot Auto Config Lib

### ISSUE THREAD

**Issue by marcosstefani** [stripe/stripe-java#1697]
### Is your feature request related to a problem? Please describe.

Hello, to make it easier and avoid having my key exposed in the code, or even searching for it via code in the environment variables, I created this auto config, if you want, I can donate them for you to keep, otherwise tell me and I will create a lib with my name in the maven repository.

https://github.com/marcosstefani/stripe-autoconfigure

### Describe the solution you'd like

 just want to know if you want this code to create a lib, otherwise I'll create it myself with my name.
If you create it, it's yours and it becomes official, otherwise there's no problem.

**Comment by richardm-stripe** [comment]
Hello @marcosstefani,

This is quite neat and I've glad you've created this. Thank you!

I could be wrong, but I think there is unfortunately no way that we can add this directly into stripe-java without requiring Spring Boot to be a dependency for all stripe-java users, whether they use Spring Boot or not, which is something we'd rather avoid. So I think it is best that, for now, you publish this in Maven under your own name.

We have deliberately kept stripe-java framework-agnostic, but the idea of providing additional libraries to officially/more deeply support frameworks like Spring Boot does come up from time to time.

Would you be willing to retitle this issue something like "Official support for Spring Boot"? We'll leave it open as a feature request to gauge interest.

**Comment by marcosstefani** [comment]
Yes No problems.  I will rename the packages to my name then.  I thought you wanted to create a new lib besides stripe-java.  In this case, you would just need to update the stripe-java version within this new lib and create a new version of this new lib when that happens.  But ok, I can create it then.  If you're interested, I wouldn't mind if you did it, but ok.  Tell me anything.

**Comment by pakrym-stripe** [comment]
Hi @marcosstefani, great idea of an integration. Unfortunately we do not have resources to maintain framework specific integrations right now.  We'd be happy if you publish the integration under your name.
